heron/scheduler-core/tests/java/com/twitter/heron/scheduler/dryrun/Constants.java

```diff
@@ -0,0 +1,25 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.twitter.heron.scheduler.dryrun;
+
+public final class Constants {
+
+  public static final String TEST_DATA_PATH =
+      "/__main__/heron/scheduler-core/tests/java/com/twitter/heron/scheduler/dryrun/testdata";
```

Comment:
I was following https://github.com/twitter/heron/blob/master/heron/common/tests/java/com/twitter/heron/common/config/Constants.java. And include the test files like [this](https://github.com/twitter/heron/pull/1629/files#diff-82363f3f748309fe4666863b76fca86fR55).

Do you have any example of what you just suggested?